test_prime.py

```diff
@@ -0,0 +1,169 @@
+# Funkce test_MetodaPravidelDelitelnosti určí některá neprvočísla,
+# pomocí toho, že je číslo končí určitým číslem,
+# které podle pravidel dělitelnosti, lze vydělit již prvočíslem.
+# (např. čísla končící na číslo 5 a 0 jdou vždy vydělit 5.
+
+
+def test_MetodaPravidelDelitelnosti():
+    assert MetodaPravidelDelitelnosti(88) is True
+    assert MetodaPravidelDelitelnosti(87) is False
+    
+# Funkce test_DeterministickaMetoda zjistuje u téhle metody,
+# jestli je číslo prvočíslo = True nebo není = False.
+
+
+def test_DeterministickaMetoda():
+    
+    assert DeterministickaMetoda(50013) is False
+    assert DeterministickaMetoda(50011) is False
+    
+# Funkce test_HeurestickaMetoda zjistuje u téhle metody,
+# jestli je číslo prvočíslo = True nebo není = False.
+    
+
+def test_HeurestickaMetoda():
+    assert HeurestickaMetoda(17) is True
+    assert HeurestickaMetoda(12) is False
+    assert HeurestickaMetoda(6) is False
+    
+# Funkce test_vstup zjistuje, jestli byl vstup zadán špatně nebo správně 
+# (správně pouze celá čísla 0,1,2,3,4,5,6,7,8,9 ne znaky a písmena to je chyba)
+
+
+'''def test_Vstup():
+    assert Vstup("1") is True
+    assert Vstup("k") is False
+'''
+
+def Vstup(Napsanecislo):
+
+    # Funkce získá vstup od uživatele proti chybám.
+
+    for znak in Napsanecislo:
+        if not (znak in "0123456789"):
+            return 0
+
+    vstupniCislo = int(Napsanecislo)
+
+    return vstupniCislo
+
+
+def Vystup(pouzitaMetoda=0):
+ 
+    # Funkce vypíše výstup programu(chybu, je či není prvočíslo; použitá metoda).
+
+    my_str1 ='CHYBA při zadávání. Zadaná hodnota neopodívá formátu,který'
+    my_str2 ='se dá přečíst.Restartuj a zadej přirozené číslo.' 
+    
+    if (pouzitaMetoda == 0):
+        print(my_str1 + my_str2)
+
+        return
+    elif (pouzitaMetoda > 0):
+        print("Zadané číslo", moznePrvocislo, "PATŘÍ mezi prvočísla.")
+    else:
+        print("Zadané číslo", moznePrvocislo, "NEPATŘÍ mezi prvočísla.")
+        pouzitaMetoda *= (-1)
+
+    print()
+
+# Popis postupů
+
+    my_str3 ='HEURISTICKÁ metoda-Rozpozná číslici 1 a vyřadí ji, číslo není.'
+    my_str4 ='prvočíslo.(číslo musí být dělitelné DVĚMA čísly, samosebou a jedničkou)'
+    my_str5 ='HEURISTICKÁ metoda-pravidla dělitelnosti.Určí neprvočísla, pomocí toho, že číslo '
+    my_str6 ='končí určitým číslem podle,pravidel dělitelnosti, lze vydělit prvočíslem.'
+    my_str7='HEURISTICKÁ metoda-Wilsonova věta. Používá matematický vztah ((n - 1)! + 1) % n'
+    my_str8 ='DETERMINISTICKÁ metoda-Zkouší pokud je číslo dělitelné některým'
+    my_str9 ='z předtím nalezených prvočísel do maximální hodnoty odmocniny'
+    my_str10 ='ze zadaného čísla.Pokud není dělitelné,je to prvočíslo.'
+    
+    if (pouzitaMetoda == 1):
+        print(my_str3 + my_str4)
+    elif (pouzitaMetoda == 2):
+        print(my_str5 + my_str6)
+    elif (pouzitaMetoda == 3):
+        print(my_str7)
+    elif (pouzitaMetoda == 4):
+        print(my_str8 + my_str9 + my_str10)
```

Comment:
upravit stejně jako u my_str1 a my_str2, tedy zrušit definici proměnné a dát do printu jako řetězce z vícero řádků